src/utils.ts

```diff
@@ -89,6 +89,10 @@ export function buildRequestApiOptions (sdk: GenesysCloudWebrtcSdk, opts: Partia
     opts.method = 'get';
   }
 
+  if (!opts.customHeaders) {
```

Comment:
Do we also need to check if we even have custom headers in the config? We aren't guaranteed to have them because its not a required config option.

Reply:
Oh that's a great point.  I'll add that in.